### PR TITLE
Update object-not-appear-in-word-2010.md

### PR DESCRIPTION
**Pull request by msbemba**
Another workaround (tested) but not sure if this fits in the correct section. I could not add a new line item for this. Please review and let me know if this fits in the document.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1450

File: Office/Client/word/object-not-appear-in-word-2010.md [hunk 1]
```diff
@@ -110,6 +110,9 @@ When you turn on the **Print drawings created in Word** option, you can view the
 3. Under **Printing options**, click to select the **Print drawings created in Word ** check box.   
 4. Click **OK**.   
 
+
+Another workaround, find the images that are no printing, **Select**. **Copy** and **Paste** them in a new word document. Now, **Select**, **Copy** and **Paste** them back in the oringal document after manually deleting the original ones.
```
Comment (JohanFreelancer9) [inline Office/Client/word/object-not-appear-in-word-2010.md:114]:
```suggestion
Another workaround is to find the images that are no printing. Select them and copy and paste them into a new word document. Then, select them again and copy and paste them back into the original document after manually deleting the originals.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by simonxjx** [comment]
Add a note before the video driver section.

**Pull request merged**